mysql-test/mroonga/wrapper/alter_table/t/back_to_innodb.test

```diff
@@ -17,6 +17,7 @@
 --source ../../../include/mroonga/skip_mysql_8_0_or_later.inc
 --source ../../../include/mroonga/skip_mariadb_10_5.inc
 --source ../../../include/mroonga/skip_mariadb_10_6.inc
+--source ../../../include/mroonga/skip_mariadb_10_7.inc
```

Comment:
Can we use `skip_mariadb_10_5_or_later.inc`?

Reply:
Oh...I see.
I've fixed it.